stargazer/APIReference/constellationRPCAPI/dag_signData.md

```diff
@@ -0,0 +1,133 @@
+---
+hide_table_of_contents: true
+---
+
+<head>
+  <meta
+    name="description"
+    content="Calculates a constellation signature of the given signature request from the selected account."
+  />
+</head>
+
+<intro-end />
+
+Calculates a constellation signature of the given signature request from the selected account.
+
+:::caution Warning
+This method adds the standard `"\u0019Constellation Signed Data:\n" + len(message) + "\n"` prefix when calculating the signature hash.
+
+The final string looks like this: `"\u0019Constellation Signed Data:\n" + len(message) + "\n" + message`
+:::
+
+:::info Note
+The only difference between `dag_signData` and `dag_signMessage` is the prefix.
```

Comment:
Ok, I finally got this because it's tricky to notice the difference between `"\u0019Constellation Signed Data:\n"` and `"\u0019Constellation Signed Message:\n"`, there is more than one difference than the just the prefix when comparing `dag_signData` and `dag_signMessage` (so we should not say there is only one difference), I'd also suggest changing the note to a warning and stating the difference between prefixes. 

Something like, "Please be sure you use the correct prefix for the correct method when verifying signatures, `dag_signData` uses 'Signed Data' while `dag_signMessage` uses 'Signed Message'".

Reply:
You're right. That's not the only difference they have. I'll update this section